pipelines/utils.nox.py

```diff
@@ -25,29 +25,38 @@
 
 from pipelines import nox
 
-TRASH = [
+DIRECTORIES_TO_DELETE = [
     ".nox",
     "build",
     "dist",
     "hikari.egg-info",
     "public",
-    ".coverage",
     ".pytest_cache",
     ".mypy_cache",
     "node_modules",
+]
+
+FILES_TO_DELETE = [
+    ".coverage",
     "package-lock.json",
 ]
 
 
 @nox.session(reuse_venv=False)
-def purge(_: nox.Session) -> None:
+def purge(session: nox.Session) -> None:
```

Comment:
this seems a bit out of scope

Reply:
I know, but it was annoying me quite a bit when it didnt work correctly

Reply:
wouldn't it be less annoying if you fixed it sooner by putting it in a pr which can be immediately pred to master instead of tagging it along with an unrelated massive change